apps/discovery_api/mix.exs

```diff
@@ -58,7 +62,7 @@ defmodule DiscoveryApi.Mixfile do
       {:phoenix_pubsub, "~> 1.0"},
       {:nanoid, "~> 2.0"},
       {:placebo, "~> 1.2.2", only: [:dev, :test]},
-      {:plug_cowboy, "~> 1.0"},
+      {:plug_cowboy, "~> 2.1"},
```

Comment:
Have we confirmed if the hosted file download bug (memory overflow) is present with this? I don't think we have enough of them for it to matter if not.